src/sentry/migrations/0025_organizationaccessrequest_requester.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2020-01-08 06:35
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations
+import django.db.models.deletion
+import sentry.db.models.fields.foreignkey
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Adding indexes to large tables. These indexes should be created concurrently,
+    #   unfortunately we can't run migrations outside of a transaction until Django
+    #   1.10. So until then these should be run manually.
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+
+    dependencies = [
+        ('sentry', '0024_auto_20191230_2052'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='organizationaccessrequest',
+            name='requester',
+            field=sentry.db.models.fields.foreignkey.FlexibleForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
+        ),
+    ]
+
+
+# BEGIN;
+# --
+# -- Add field requester to organizationaccessrequest
+# --
+# ALTER TABLE "sentry_organizationaccessrequest" ADD COLUMN "requester_id" integer NULL;
+# ALTER TABLE "sentry_organizationaccessrequest" ALTER COLUMN "requester_id" DROP DEFAULT;
+# CREATE INDEX "sentry_organizationaccessrequest_573f8683" ON "sentry_organizationaccessrequest" ("requester_id");
```

Comment:
We need concurrently for the index.

Reply:
This table is 332 rows, is it important? 

At the moment, we can't create concurrently in a migration since it runs in a transaction, and so people will need to run it manually instead. Post 1.10 we can set `atomic = False` and then create indexes concurrently.

Reply:
I guess it sonly a few hundred rows, so _should_ be fine, but just wanted to point it out since this wasn't mentioned.

Reply:
Yeah, hopefully we can do something to make this automatic post 1.10, or at least easier to remember.

Reply:
Yeah, it should be ok. I'd just like to see at least this consciously acknowledged so we're aware this is a decision rather than a lack of decision.

Reply:
Thanks, I didn't mark it as dangerous due to the row count, but I'm good with what you think is best